app/config.js

```diff
@@ -56,8 +56,16 @@ module.exports = require('rc')('agregore', {
   hyperOptions: {
     storage: DEFAULT_HYPER_DIR
   },
-  // All options here: https://github.com/ssbc/ssb-config#configuration
-  ssbOptions: {},
+  /**
+   * All ssb options here: https://github.com/ssbc/ssb-config#configuration
+   * For bundled ssb server we use ssbd. Pass in an array of require'd ssb plugins
+   */
+  ssbOptions: {
+    appname: DEFAULT_SSB_APPNAME,
+    ssbd: {
+      plugins: require('@metacentre/shipyard-ssb')
```

Comment:
Since these settings need to be JSON-compatible, could we have the plugins be an array of strings instead and do the require within ssb-protocol.js?

```suggestion
      plugins: ['@metacentre/shipyard-ssb']
```

Reply:
The node resolution algorithm makes this painful to do. I was doing exactly this (loading from bare specifier strings) in shipyard and found while it did work the user experience wasn't great. Kinda violated the principle of least surprise. 

So long as I installed all my ssb plugins in `~/node_modules` life was good. That's why I initially wrote ssbd to only consume already require'd modules. But of course `.agregorerc` is json so I had to put the requiring of strings into ssbd. 

In my opinion, since agregore is bundling an ssb daemon, the simplest thing is to require plugins in agregore for the default case of new users. 

For users who are already on ssb and want to choose their own list of ssb plugins, they can do that in `.agregorerc`:

```json
"ssbOptions": {
    "ssbd": {
      "plugins": ["/absolute/path/to/list/of/plugins/to/require"]
    }
  }
```

Clearly I need to document this!

Setting the `.agregorerc` file could be done from `agregore://settings/ssb` for a smooth user experience.